pkg/nsx/services/mediator/mediator.go

```diff
@@ -67,8 +71,33 @@ func (serviceMediator *ServiceMediator) GetAvailableSubnet(subnetSet *v1alpha1.S
 			return *nsxSubnet.Path, nil
 		}
 	}
+	var tags []model.Tag
+	findLabelDefaultPodSubnetSet := false
+	for k, v := range subnetSet.Labels {
+		if k == common.LabelDefaultSubnetSet && v == common.LabelDefaultPodSubnetSet {
+			findLabelDefaultPodSubnetSet = true
+			break
+		}
+	}
+	namespace := &corev1.Namespace{}
+	namespacedName := types.NamespacedName{
+		Name: subnetSet.Namespace,
+	}
+	if err := serviceMediator.SubnetService.Client.Get(context.Background(), namespacedName, namespace); err != nil {
+		return "", err
+	}
+	namespace_uid := namespace.UID
+	if findLabelDefaultPodSubnetSet {
+		tags = append(tags,
+			model.Tag{Scope: common.String(common.TagScopePodSubnetNamespaceUID), Tag: common.String(string(namespace_uid))},
+			model.Tag{Scope: common.String(common.TagScopePodSubnetNamespace), Tag: common.String(subnetSet.Namespace)})
+	} else {
+		tags = append(tags,
```

Comment:
From https://github.com/vmware-tanzu/nsx-operator/pull/304, TagScopeNamespaceUID and TagScopeNamespace will become the basic tags for all subnet in the builder basicTags, if that is true, we might need to add two new tags for VM subnet. if we want to use TagScopeNamespaceUID and TagScopeNamespace only for VM subnet, we need to align with Tao's change in basic tags.

Reply:
TagScopeNamespaceUID TagScopeNamespace for VM subnet
TagScopePodSubnetNamespaceUID TagScopePodSubnetNamespace for pod subnet
right？

Reply:
In order to align with ncp, I suggest, TagScopeNamespaceUID TagScopeNamespace for pod subnet.
TagScopeVMSubnetNamespaceUID nsx-op/vm_subnet_namespace_uid
and TagScopeVmSubnetNamespace: nsx-op/vm_subnet_namespace
These two tags for vm subnet.

we note https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/util/utils.go#L368 subnetset also have TagScopeNamespaceUID and TagScopeNamespace, since subnetset tags will be applied all subnets managed by
subnetset, so, we also need to remove TagScopeNamespaceUID and TagScopeNamespace from subnetset basic tags in util.BuildBasicTags.

you can submit a new patch later when this patch merged:
1. remove TagScopeNamespaceUID and TagScopeNamespace  from subnet in  https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/util/utils.go#L353 in the util.BuildBasicTags.
2. remove TagScopeNamespaceUID and TagScopeNamespace  from subnetSet in  https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/util/utils.go#L368 in the util.BuildBasicTags.

Reply:
OK, add TagScopeVMSubnetNamespaceUID and TagScopeVMSubnetNamespace, others would be refactored in next patch.